Server/src/cli/commands/animation.py

```
@@ -0,0 +1,85 @@
+    "--layer", "-l",
+    default=0,
+    type=int,
+    help="Animator layer."
+)
+@click.option(
+    "--search-method",
+    type=click.Choice(["by_name", "by_path", "by_id"]),
+    default=None,
+    help="How to find the target."
+)
+def play(target: str, state_name: str, layer: int, search_method: Optional[str]):
```

Ruff (0.14.10)

Unused function argument: `layer`

(ARG001)